README.md

```diff
@@ -42,6 +44,15 @@ The services of a client divide the API into logical chunks and correspond to
 the structure of the GitHub API documentation at
 https://developer.github.com/v3/.
 
+
+NOTE: The [context](https://godoc.org/context) package is being used to provide
+request scoped values to the API calls. If case there is no context available,
+then `context.Background()` can be used as a starting point.
```

Comment:
> This is not an accurate statement.

I'm sorry about the first statement;  I should be researched better. I learned about the importance and usage of the `context` package only recently after pushing the changes.

How about we rephrase the comment as follows:

> Using the `context` package, one can easily pass cancelation signals and deadlines to various services of the client for handling a request. In case there is no context available, then `context.Background()` can be used as a starting point.

Reply:
Sounds good.